### ISSUE TITLE
Error NOT NULL constraint upon canvas installation

### ISSUE THREAD

**Issue by balkalabs** [cnvs/easel#102]
To avoid installation error like 
```
SQLSTATE[23502]: Not null violation: 7 ERREUR:  une valeur NULL viole la contrainte NOT NULL 
```
when doing the command 
```
php artisan canvas:install
```

Please upgrade \src\Console\Commands\CanvasCommand createUser from : 
```
    protected function createUser($email, $password, $firstName, $lastName)
    {
        $user = User::firstOrCreate(['email' => $email]);
        $user->password = bcrypt($password);
        $user->first_name = $firstName;
        $user->last_name = $lastName;
        $user->display_name = $firstName.' '.$lastName;
        $user->role = CanvasHelper::ROLE_ADMINISTRATOR;
        $user->save();

        $this->author($user->display_name);
        $this->comment('Saving admin user account...');
        $this->progress(1);
    }
```
to
```
    protected function createUser($email, $password, $firstName, $lastName)
    {
        $user = User::firstOrCreate([
            'email' => $email,
            'password' => bcrypt($password),
            'first_name' => $firstName,
            'last_name' => $lastName,
            'display_name' => $firstName.' '.$lastName
        ]);
        $user->role = CanvasHelper::ROLE_ADMINISTRATOR;
        $user->save();

        $this->author($user->display_name);
        $this->comment('Saving admin user account...');
        $this->progress(1);
    }
```

**Comment by austintoddj** [comment]
Thanks for sending this over @balkalabs. 

Looks good by me, @reliq?

**Comment by reliq** [comment]
@austintoddj will have to take some time to iron out this issue properly,  this fix could cause issues on some systems

**Comment by DanielFurmanov** [comment]
@austintoddj I believe this was fixed in https://github.com/cnvs/easel/pull/117

**Comment by austintoddj** [comment]
@DanielFurmanov Great thanks for giving the heads up. @balkalabs Can you confirm that this was fixed in #117 please?
